src/browser/base/content/zen-styles/zen-tabs/vertical-tabs.css

```diff
@@ -367,8 +365,15 @@
 
     & #tabbrowser-arrowscrollbox-periphery {
       & > toolbarbutton {
+        padding: var(--toolbarbutton-inner-padding);
+
+        & image {
+          height: 16px;
+          width: 16px;
+          padding: 0 !important;
+        }
+
         margin: 0 auto !important;
-        padding: 0 !important;
```

Comment:
Are you sure about this?

Reply:
This was done to fix an issue with the `newtab-button` icon being offside from the tab icons.
Before:
![image](https://github.com/user-attachments/assets/d8c42d98-dd9b-48cf-aba1-99b3eaff4ead)

After:
![image](https://github.com/user-attachments/assets/57309cf6-7db9-4f6f-b7e2-98b52c337fe4)